components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/impl/APIUsageStatisticsRestClientImpl.java

```diff
@@ -1067,13 +1067,13 @@ public ApiTopUsersListDTO getTopApiUsers(String apiName, String version, String
         try {
             StringBuilder lastAccessQuery = new StringBuilder("from " + tableName);
             if (!providerName.startsWith(APIUsageStatisticsClientConstants.ALL_PROVIDERS)) {
-                lastAccessQuery.append(" on(" + APIUsageStatisticsClientConstants.TENANT_DOMAIN + "=='" + tenantDomain
+                lastAccessQuery.append(" on(apiCreatorTenantDomain=='" + tenantDomain
```

Comment:
why we need to remove the constant and place a variable?

Reply:
that constant has a different value